### DISCUSSION TITLE
First-party cookie with samesite=none

### DISCUSSION THREAD

**Discussion by helenafranca** [GoogleChromeLabs/ps-analysis-tool#630] [Support Forum] [answered]
Will a first-party cookie with samesite=none work after third-party cookie deprecation?

![Screenshot 2024-04-17 10 55 36 AM](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/85519105/088fbcf3-0993-4b5e-a693-ba4e49ce9a44)

**Comment by amedina** [comment, marked as answer]
If the cookie is 3P and the `SameSite` attribute is set to `None`, it will be blocked. In the example shown, the corresponding cookie is 1P.
